judge/views/contests.py

```diff
@@ -167,6 +167,7 @@ def get_context_data(self, **kwargs):
         context['meta_description'] = self.object.summary or metadata[0]
         context['og_image'] = self.object.og_image or metadata[1]
         context['has_moss_api_key'] = settings.MOSS_API_KEY is not None
+        context['logo_override_image'] = self.object.logo_override_image
```

Comment:
It might be nice if (a) the contest is private to a single organization and (b) that organization has a logo override and (c) the contest doesn't override it, to use the organization's.